zerver/context_processors.py

```diff
@@ -86,6 +86,8 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
     if settings.ZILENCER_ENABLED:
         apps_page_url = '/apps/'
 
+    apps_page_web = 'https://zulipchat.com/accounts/go/'
```

Comment:
This should use `ROOT_DOMAIN_URI` rather than hardcoding zulipchat.com.

Reply:
Done @timabbott